### PR TITLE
add appveyor for building incremental zips

### PR DESCRIPTION
**Pull request by L0laapk3**
~~WIP~~
Done

### CODE REVIEW

**Comment by VirxEC** [comment]
Once this is done, I'll be able to finalize my code that will download only the updated files in the botpack. Instead of downloading a 70MB zip, most updates will be 5-30KB.

**Comment by L0laapk3** [comment]
@tarehart While this could work from any github account, it may not be a bad idea to set up a github account solely for the purpose of rolling out these incrementals so that it only has access to the RLBotPack repo. A linked appveyor account is also required. I can walk you trough setting them up, or I can do it myself, whichever you prefer.

(Hopefully this doesn't happen) but if something goes wrong in the future it might be useful that I have access to those accounts.

**Comment by L0laapk3** [comment]
Note for future reference: I have created a branch where is demonstrated how to change appveyors "memory" of what the latest version number and commit hash was: https://github.com/L0laapk3/RLBotPack/blob/appveyorfix/.appveyor.yml . Could be helpful in case something goes wrong in the future.

File: .appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
+image: Visual Studio 2019
+platform:
+   - amd64
+
+environment:
+   APPVEYOR_API_TOKEN:
+      secure: qTt+0485sfAXM2Sz+QLZ+jjziNf4One8/A062QFN1a4=
+   GITHUB_API_TOKEN:
+      secure: TpVl9vErdKkXR/RUlVtUDfeITAn5BtOhG3JxiFJRGp85c3B7boCFf6CDOxNgMr/m
+   DEPLOY_BRANCH: 'master'
```
Comment (tarehart) [inline .appveyor.yml:10]: I think these are encrypted based on a particular appveyor account - https://www.appveyor.com/docs/build-configuration/#secure-variables

I assume these values would need to be updated once we decide what accounts will be used?

Comment (L0laapk3) [reply .appveyor.yml:10]: Correct. For github the key needs `repo` permissions. I can walk you trough creating and setting everything up once we're there

**Comment by L0laapk3** [comment]
I recommend to create a separate github account for this that only has access to rlbotpack and no other repo's.
An appveyor account is also needed, for future ease of use its probably a good idea to use the before mentioned github account as login for the appveyor.

The tokens:
**The tokens are both to be encrypted trough https://ci.appveyor.com/tools/encrypt**
- Github token is generated at https://github.com/settings/tokens and needs `repo` permissions as mentioned
- Appveyor token is generated at https://ci.appveyor.com/api-keys

**Review by tarehart — APPROVED** [review]

**Pull request merged**